popup.js

```diff
@@ -237,27 +237,15 @@ async function render_plan() {
 
     // Display plan name
     $plan.innerHTML = plan.plan;
-    if (plan.plan === 'free') {
-        if (settings.key !== '') {
-            $incorrect_key.classList.remove('hidden');
-        }
-        else {
-            $incorrect_key.classList.add('hidden');
-        }
-        $plan.classList.remove('green');
-        $plan.classList.add('red');
-    }
-    else {
-        $incorrect_key.classList.add('hidden');
-        $plan.classList.remove('red');
-        $plan.classList.add('green');
-    }
 
-    if(plan.plan.includes("Banned")) {
-        $ipbanned_warning.classList.remove('hidden');
-    } else {
-        $ipbanned_warning.classList.add('hidden');
-    }
+    const freePlan = plan.plan.toLowerCase().includes('free');
```

Comment:
An alternative is:

```suggestion
    const planIs = plan.plan.toLowerCase().includes
```

Then below, for example:

```
    $incorrect_key.classList.toggle('hidden', !planIs('free') || settings.key !== '')
```